commands/service_dev.go

```diff
@@ -51,8 +51,11 @@ func (c *serviceDevCmd) preRunE(cmd *cobra.Command, args []string) error {
 
 func (c *serviceDevCmd) runE(cmd *cobra.Command, args []string) error {
 	statuses := make(chan provider.DeployStatus)
-	go printDeployStatuses(statuses)
+	statusPrintDone := make(chan struct{})
```

Comment:
Use sync.Waitgroup as it's perfect for sync. goroutines (it applies everywhere where done channel is used)

```go
var wg sync.WaitGroup
wg.Add(1)
go func() {
  printDeployStatuses()
  wg.Done()
}()

Deploy()
wg.Wait()
```

Reply:
done chans are pretty common and really simple to use. could you create a proposal for this on forum by referencing this PR please?

Reply:
this is a good argument, changes will be made 👍